dataeng/resources/answer-distribution.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+if [ -z "$NUM_REDUCE_TASKS" ]; then
+    NUM_REDUCE_TASKS=$(( $NUM_TASK_CAPACITY * 2 ))
+fi
+NUM_REDUCE_TASKS=$(( $NUM_REDUCE_TASKS * 2 ))
+
+now=$(date +%H)
+
+name=hourly
+src="$SOURCES"
+dest=${DESTINATION_PREFIX}/$now
+if [ -z "$MANIFEST_PREFIX" ]; then
+    MANIFEST_PREFIX="$DESTINATION_PREFIX"
+fi
+manifest=${MANIFEST_PREFIX}/$now/manifest.txt
+
+# Remove any previous intermediate output for this hour
+pip install awscli
+aws s3 rm --recursive $dest || true
+aws s3 rm $manifest || true
```

Comment:
Normally $manifest is under $dest, but in the -release version of the job it is not.  It should not hurt to attempt to delete the file twice anyway.

Reply:
I don't remember the specific ways in which the different jobs have different ways of structuring their intermediate output, but I don't think we need to go out of our way to preserve it, especially for the dev-edx.cfg version.  (I'm not sure if that's what you mean by -release.)   I expect there is one way that is more sensible, and I would hope it is the production setup.   Ultimately it's all currently going to edx-analytics-scratch bucket anyway, which is not consistent with other jobs.  Keep in mind that this was the first job we ever built, back four plus years ago.   So don't work too hard to preserve the idiosyncracies. : )